doc/source/user_guide/scale.rst

```diff
@@ -0,0 +1,404 @@
+.. _scale:
+
+*************************
+Scaling to large datasets
+*************************
+
+Pandas provides data structures for in-memory analytics, which makes using pandas
+to analyze larger than memory datasets somewhat tricky.
```

Comment:
This document is not only for "larger than memory" data right? It becomes already tricky if your dataset is (some factor) smaller than your memory, right? (because we create copies, because reading can take more memory, ...)

At least the first sections in this document equally apply as performance considerations on smaller-than-memory datasets

Reply:
Tried to clarify this a bit (in part by removing the "use efficient file formats" section.